coda/coda_mdstore/presentation.py

```diff
@@ -86,8 +85,7 @@ def bagSearch(bagString):
     return a list of related bags
     """
 
-    bagList = Bag.objects.filter(
-        bag_info__field_body__search=bagString).distinct()
+    bagList = Bag.objects.filter(bag_info__field_body__exact=bagString).distinct()
```

Comment:
I know the built-in `__search` lookup has been removed, and I see that there is an error if you try to use the replacement lookup you defined in this PR, but I don't think using the `__exact` lookup here is going to work for our use case. I believe it changes the behavior of the searching to require the user to, for instance, need to know the full description content of a bag and search for that in its entirety rather than a few keywords.

Reply:
Oh..I intended to use `__search` here. It is here by mistake. Can you tell me more about the error with the replacement lookup?

Reply:
Yeah, actually there isn't an error! I was messing with the code locally the other day and forgot and had caused the error. I think we just need to switch to use __search here like you intended.